src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,19 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
     //write your code here
+    static final int VALID_AGE = 35;
+    static final int VALID_LIVE_IN_UKRAINE = 10;
```

Comment:
```suggestion
    private static final int VALID_AGE = 35;
    private static final int VALID_LIVE_IN_UKRAINE = 10;
```

don't forget about access modifiers